src/Assignment-5/assignment5.css

```diff
@@ -1,7 +1,77 @@
-/* Our CSS file. */
 
+html, body {
+  height: 120%;
+}
+body {
+  background: #eee;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-direction: column;
+  font-family: -apple-system,BlinkMacSystemFont,"SegoeUI",Roboto,Oxygen-Sans,Ubuntu,Cantarell,"Helvetica Neue",sans-serif;
+}
+.wrap {
+  height: 100%; 
+}
+.card {
+  display: flex;
+  flex: 0 0 auto;
+  background: #fff;
+  max-width: 700px;
+  margin: 100px 0;
+}
 
-.container, .content {
-width: 100%;
-height: 100%;
+.pic-details{
```

Comment:
the picture background color should be different something like green and also the image should overflow your assignment doesn't  full-fill this criteria